### PR TITLE
Adds detection for Opera GX browser

### PR DESCRIPTION
**Pull request by liviuconcioiu**

### CODE REVIEW

File: regexes/client/browsers.yml [hunk 1]
```diff
@@ -5,6 +5,19 @@
...
 ###############
 
+# Opera GX (https://www.opera.com/gx#start | http://ftp.opera.com/pub/opera_gx/)
+- regex: 'OPRGX(?:/(\d+[\.\d]+))?'
+  name: 'Opera GX'
+  version: '$1'
+  engine:
+    default: 'Blink'
+
+- regex: '(?:OPR/(60.0.3255.(50747|50962|51199)|62.0.3331.(105|117|132|52|81|96)|63.0.3368.(54789|54979|55362|55666|56078|56786|57388|57756|58152|66|71)|64.0.3417.(100|121|139|146)))'
```
Comment (sgiehl) [inline regexes/client/browsers.yml]: Would you mind mentioning where you found the information that those specific versions are Opera GX only and can't be a normal Opera browser?

Comment (liviuconcioiu) [reply regexes/client/browsers.yml]: [http://ftp.opera.com/pub/opera_gx/](http://ftp.opera.com/pub/opera_gx/) I have added the link to regexes/client/browsers.yml file.

Comment (sgiehl) [reply regexes/client/browsers.yml]: I see the list of versions there. But I wonder if a normal opera might not have the same version number?

Comment (liviuconcioiu) [reply regexes/client/browsers.yml]: From what I see here [http://ftp.opera.com/pub/opera/desktop/](http://ftp.opera.com/pub/opera/desktop/), it seems that they use the same version numbers for Opera too :(

Comment (sgiehl) [reply regexes/client/browsers.yml]: Then lets remove the version specific detection of Opera GX

**Review by sgiehl — APPROVED** [review]

**Pull request merged**